app/src/main/java/com/futurice/freesound/feature/home/HomeFragmentDataEvents.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.feature.home
+
+import io.reactivex.Observable
+import java.util.concurrent.TimeUnit
+
+// This could probably just be a file, it would nice to have symmetric way of doing this
+internal class HomeFragmentDataEvents(private val userDataModel: UserDataModel) {
+
+    fun dataEvents(): Observable<Fragment.DataEvent> {
+        return userDataModel.homeUser
+                .delay(5, TimeUnit.SECONDS) // for testing purposes!!!!
+                .map { Fragment.DataEvent.UserDataEvent(it) as Fragment.DataEvent }
+                .toObservable()
```

Comment:
The general pattern would be for the VM to take an `Observable<DataEvents>`, in this case it's only a `Single`, but I wanted to try and emulate the more general case.